spec/factories/users.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :user do
+    
```

Comment:
@hcbviolet since Devise is an Engine, you do not need to test it. Nix this please!